### PR TITLE
WestMidlands Class 3 - Abiya Yaccobu - Git/GitHub - week 2

### PR DESCRIPTION
**Pull request by AbiyaY**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

-----
[View rendered .github/pull_request_template.md](https://github.com/AbiyaY/GitHomeworkFixErrors/blob/main/.github/pull_request_template.md)
[View rendered homework.md](https://github.com/AbiyaY/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-You delete it
+A fork is a copy of a repository. It allows you to freely experiment with changes without affecting the original project.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+when you clone a repo, you make a copy of the complete history.
```
Comment (garvey-marcus-code) [inline homework.md:67]: I would also add that cloning makes a copy on your local device.

**Review by garvey-marcus-code — COMMENTED** [review]
I've completed my review.

File: homework.md [hunk 2]
```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
...
 
-Send you code to Github
+The commit command is used to save your changes to the local repository.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+HTML, CSS, JavaScript.
```
Comment (Maria-Chira) [inline homework.md]: I think HTML, CSS and JavaScript are not programs.  Can you please check this answer again?

**Review by Maria-Chira — COMMENTED** [review]
I have finished my review.

**Pull request closed without merging**